user-guide/query-examples-hacks/sunbusrt-sankey-example.md

```diff
@@ -0,0 +1,105 @@
+# Funnel Queries (Sankey and Sunbusrt Visualizations)
+
+Sankey and Sunburst visualizations are very good for understanding behavioral flows and sequences.
+
+To create such a chain of events you'll need to define the actions/event you want to examine, define how many events are in a sequence (max number of events) and assign chronological session numbers per user.
```

Comment:
Sankey and Sunburst visualizations are very good for understanding behavioral flows and sequences you want to examine and create sessions out of them. - ?